colaraz/cms/static/js/header.js

```diff
@@ -116,3 +120,72 @@ function removeNotificationsCount() {
         markNotificationsAsRead();
     }
 }
+
+function getAndPopulateJobAlerts() {
+    $.ajax({
+        type: "GET",
+        url: colarazJobAlertsFetchingUrl,
+        success: function (resp) {
+            let job_alerts = "";
+            if (resp.d.length) {
+                resp.d.forEach(element => {
+                    job_alerts += createJobAlert(element.Image, element.Heading, element.Message, element.RelativeTime);
+                });
+            }else{
+                job_alerts = "<li><p>No new job alerts</p></li>"
+            }
+
+            handleJobAlertsListing(job_alerts);
+            handleJobAlertsCount(resp.d.length);
+
+        },
+        error: function (resp) {
+            console.error(`Job Alerts API gave following error: ${resp.Message}`);
+        },
+    });
+
+    function handleJobAlertsListing(job_alerts) {
+        $("#job-alerts-list").html(job_alerts);
+    }
+    function handleJobAlertsCount(count) {
+        if (count == 0) {
+            let countSpan = $("#unread-job-alerts-count");
+            countSpan.removeClass("count");
+            countSpan.html("");
+        } else {
+            let countSpan = $("#unread-job-alerts-count");
+            countSpan.addClass("count");
+            countSpan.html(count);
+        }
+    }
+
+    function createJobAlert(img_src, heading, message, relativeTime) {
+        return `<li>` +
+                    `<div class="media">` +
+                        `<a href="#">` +
+                            `<img src="${img_src ? img_src : "https://placehold.it/34x34"}" alt="">` +
+                        `</a>` +
+                    `</div>` +
+                    `<div style="font-size: 12px;padding: 0;color: #333;" class="description">` +
```

Comment:
We can also create multiline strings like this...
```
return `
abc
abc
`
```

Reply:
I didn't know that, thanks :)
#done